test/integration/index.js

```diff
@@ -1,4 +1,4 @@
-describe('Integration tests', function() {
+describe('Integration tests', () => {
```

Comment:
everything in /test/ will need to use `function() {}` instead of arrow functions.
https://mochajs.org/#arrow-functions